superset/.s2i/bin/assemble

```diff
@@ -27,7 +27,11 @@ cp -Rf /tmp/src/. ./
 echo "---> Building application from source..."
 # TODO: Add build steps for your application, eg npm install, bundle install, pip install, etc.
 pip install --no-cache-dir \
-    apache-superset[trino,postgres,mysql,db2,thumbnails]==1.1.0 \
-    Authlib==0.15.4
+    Authlib
```

Comment:
This throws an error because of the missing `\` line continuation

Reply:
good catch, just fixed it.

(it would be awesome if such problem can be caught during CI so that it will be visible automatically)